recipes/snirf/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "snirf" %}
+{% set version = "0.7.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/snirf-{{ version }}.tar.gz
+  sha256: bf214ff1bbad2a9971efd54cb696fa49e4f185a65578024be2379210e4ebcaa9
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - h5py >=3.1.0  # needed for import in setup.py
+  run:
+    - python >=3.6
+    - h5py >=3.1.0
+    - numpy
+    - setuptools
+    - pip
```

Comment:
These are odd requirements for run but it is in the metadata. Maybe open an issue upstream?